editor/src/components/canvas/__snapshots__/ui-jsx-canvas.spec.tsx.snap

```diff
@@ -257,30 +257,11 @@ Object {
       "data-uid": "aaa scene-aaa",
       "skipDeepFreeze": true,
       "style": Object {
-        "alignContent": undefined,
```

Comment:
where did these undefineds go? it _looks like_ this style object is only populated from the spy and not the dom walker

Reply:
:thinking:

Reply:
Fixed by the removal of the fragment in the Scene Root, but oh my do we need to remove that pragma now :grimacing: